enabler/src/de/schildbach/pte/AbstractNavitiaProvider.java

```diff
@@ -845,10 +845,11 @@ public SuggestLocationsResult suggestLocations(final CharSequence constraint) th
 
             if (head.has("places")) {
                 final JSONArray places = head.getJSONArray("places");
+                final int priorityBase = places.length();
 
                 for (int i = 0; i < places.length(); ++i) {
```

Comment:
Just a small thing, but you could use the introduced variable here too. And maybe `numPlaces` would be a more general name?

Reply:
Fixed